### DISCUSSION TITLE
What is the point of snaphots?

### DISCUSSION THREAD

**Discussion by zax-ftw** [mbnuqw/sidebery#1472] [Q&A]
I'm using this add-on for several days already and other than moving tabs between (wrong) panels occasionally it remembers panels, tabs and tabs trees when I restart the Firefox (I have "open previous windows and tabs" enabled since forever). So other than some backup purposes or moving tabs between devices, what is the intention here?

**Comment by albino1** [comment]
Backing up tab states, restoring when things go wrong, returning to a prior point in time to try and find the tab you can't remember.  There's plenty of good reasons to keep backups.  There's very popular entire addons devoted to tab state backups (i.e. Tab Session Manager), so the fact that Sidebery includes it is a really nice feature IMHO.

**Reply by zax-ftw** [reply]
Indeed, its awesome as additional feature but I was somewhat under the impression that people spam `snapshot` button on exit to revert from snapshot in next session? What seems to be completely unnecessary (correct me if I'm missing something) because previous session gets loaded by default automatically?

**Reply by albino1** [reply]
I'm not sure how other people are using it, but personally, I have snapshots set to be created automatically every hour.  I also make a manual snapshot whenever I update Firefox, update Sidebery, or in the rare instances where I reboot or close Firefox.  All for safety in my case, as you don't want to potentially lose hundreds or thousands of tabs and their organizational structure, especially if you have them sorted into specific trees, panels, and containers.
